Ome/JavaScript/Assignment 2/Q1.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JavaScript - Find second largest number</title>
+    <style>
+        body {
+            text-align: center;
+            align-items: center;
+            align-content: center;
+        }
+
+        .container-box {
+            align-items: center;
+            border: 2px solid green;
+            width: 25%;
```

Comment:
increase the width of the box or decrease the size of the input type so that they look alligned

Reply:
Correrction done, Please check